sand/extensions/search/aggregated_search.py

```diff
@@ -0,0 +1,34 @@
+from typing import List, Union
+from sand.extension_interface.search import IEntitySearch, IOntologySearch
+from sand.models.search import SearchResult
+
+
+class AggregatedSearch(IEntitySearch, IOntologySearch):
+
+    def __init__(self):
+        self.search_objs = []
+
+    def add(self, search_obj: Union[IEntitySearch, IOntologySearch]):
+        """adds a search object to the aggregated search"""
+        self.search_objs.append(search_obj)
```

Comment:
we need two different kind of search_objs, 1 for IEntitySearch, and 1 for IOntologySearch.

Reply:
added separate lists.